shacl12-node-expr/index.html

```diff
@@ -614,19 +775,103 @@ <h3>Literal Expressions</h3>
 
 			<section id="blank-node-functions">
 				<h2>Node Expressions based on Blank Nodes</h2>
-				<p>
-					...
-				</p>
+				<section id="NamedParameterFunctions">
+					<h3>Named Parameter Functions</h3>
+					<p>
+						A <dfn>named parameter function</dfn> is a <a>node expression function</a>
+						that is represented by a <a>blank node</a> that is the <a>subject</a> of at least
+						one <a>triple</a> where the <a>predicate</a> can be used to uniquely identify the function,
+						which is known as the <dfn>key parameter</dfn>.
+					</p>
+<p>The evaluation of a <a>named parameter function</a> can produce either:
+<ul>
+<li>zero <a>output nodes</a>, i.e. the empty list</li>
+<li>one or more <a>output nodes</a>, i.e. a list of one or more nodes</li>
+<li>an <a>evaluation failure</a>, i.e. an (unexpected) error during the evaluation</li>
+</ul>
+</p>
```

Comment:
HTML forbids `<ul>` or `<ol>` inside a `<p>`.
```suggestion
					<p>
					  The evaluation of a <a>named parameter function</a> can produce any of the following:
					</p>
					<ul>
					  <li>zero <a>output nodes</a>,
					    i.e., an empty list</li>
					  <li>one or more <a>output nodes</a>,
					    i.e., a list of one or more nodes</li>
					  <li>an <a>evaluation failure</a>,
					    i.e., an (unexpected) error during the evaluation</li>
					</ul>
```